draft-ietf-quic-transport.md

```diff
@@ -920,28 +920,29 @@ A STREAM frame is shown below.
  0                   1                   2                   3
  0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
 +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|       [Data Length (16)]      |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
 |                    Stream ID (8/16/24/32)                   ...
 +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
 |                Offset (0/16/24/32/40/48/56/64)              ...
```

Comment:
Yeah, that's a separate issue.  I can understand why it is that way.  That is, there isn't much value in an 8-bit offset given that you are likely to cram > 8 octets in a single packet, which is the most common case for needing offsets.  That said, I think that we would want to discuss what options make sense.

I can see arguments for 0, 16, and 32 and 64 would avoid the need for rollover calculations.  The other options seem like overoptimizing. varint encoding would be preferable in that case.